index.html

```diff
@@ -1,286 +1,430 @@
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <div class="name-form">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              placeholder="First name"
+              maxlength="30"
+              required
+            />
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              placeholder="Last name"
+              maxlength="30"
+              required
+            />
+          </div>
+          <input
+            type="email"
+            id="email"
+            name="_replyto"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+          ></textarea>
+          <button type="submit" class="link centered-link">Send</button>
+        </form>
+      </div>
+    </main>
+    <footer class="social footer-social">
+      <a
+        class="social-link"
+        href="https://github.com/SpaYco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/github.svg"
+          alt="GitHub Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://www.linkedin.com/in/spayco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/linkedin.svg"
+          alt="LinkedIn Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://angel.co/spayco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/angel.svg"
+          alt="AngelList Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://twitter.com/iSpaYco"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/svg/twitter.svg"
+          alt="Twitter Icon"
+        />
+      </a>
+      <a
+        class="social-link"
+        href="https://spayco.medium.com/"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <img
+          class="social-icon"
+          src="./media/png/xicon.png"
+          alt="Medium Icon"
+        />
+      </a>
+    </footer>
+    <div class="nav-overlay off-display">
+      <div class="nav-mobile-container">
+        <div class="xicon-image"><img src="/media/png/xicon.png" alt="Exit Icon"></div>
+        <ul class="overlay-menu d-flex">
+          <li class="overlay-link"><a href="#works">Portoflio</a></li>
+          <li class="overlay-link"><a href="#about">About</a></li>
+          <li class="overlay-link"><a href="#contact">Contact</a></li>
         </ul>
```

Comment:
[OPTIONAL] Since a `ul` tag doesn't have a focus state which helps for accessibility, to make this hamburger icon accessible I suggest you add a `tab-index `attribute so users who rely on the keyboard can highlight it by pressing the **tab key**. Also, consider adding the `aria-controls` attribute to make it even more accessible to users that depend on screen readers